<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pv_config_dialog</class>
 <widget class="QDialog" name="pv_config_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gb_load">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Load Config</string>
   </property>
   <widget class="QLineEdit" name="le_file_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid black;
border-radius:3;

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn__load_browse">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="btn_load_accept_reject">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_create_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>451</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Create/Edit Config</string>
   </property>
   <widget class="QPushButton" name="btn_new_config">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>New Config</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid black;
border-radius:3;

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit_browse">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="btn_edit_accept_reject">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_load_accept_reject</sender>
   <signal>accepted()</signal>
   <receiver>pv_config_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_load_accept_reject</sender>
   <signal>rejected()</signal>
   <receiver>pv_config_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
